src/test/util/setup_common.cpp

```diff
@@ -141,7 +141,7 @@ BasicTestingSetup::BasicTestingSetup(const ChainType chainType, TestOpts opts)
     const std::string test_name{G_TEST_GET_FULL_NAME ? G_TEST_GET_FULL_NAME() : ""};
     if (!m_node.args->IsArgSet("-testdatadir")) {
         // By default, the data directory has a random name on each test run
-        const auto rand_str{g_rng_temp_path.rand256().ToString()};
+        const auto rand_str{util::ToString(g_rng_temp_path.rand32())};
```

Comment:
nit: for awareness, may want to mention in OP that this changes directory names from 256-bit randoms to 32-bit randoms. I'm not seeing an issue with it (virtually non-existent likelihood of collision).